### PR TITLE
[CDF-976]  CCC - Rows with the same value generate its incorrect aggr…

### PR DESCRIPTION
**Pull request by stanislau-strelchanka**
[CDF-976]  CCC - Rows with the same value generate its incorrect aggregation

### CODE REVIEW

**Comment by stanislau-strelchanka** [comment]
@dcleao, @pamval  please review

File: package-res/cdo/datum.js [hunk 1]
```diff
@@ -52,6 +55,21 @@ function(data, atomsByName) {
 
     isInterpolated: false,
     interpolation: null, // type of interpolation
+
+    generateKey: function(atom, keySep, index) {
+        if(atom.dimension && atom.dimension.isKey) {
```
Comment (dcleao) [inline package-res/cdo/datum.js]: I don't think that there is a case where an atom does not have a dimension.

File: package-res/cdo/meta/dimensionType.js [hunk 2]
```diff
@@ -316,6 +316,8 @@ function(complexType, name, keyArgs) {
      * @internal
      */
     this._format = format || null;
+
+    this.isKey = def.get(keyArgs, "isKey");
```
Comment (dcleao) [inline package-res/cdo/meta/dimensionType.js]: Should normalize to boolean (e.g. use `!! .` or `Boolean( . )`)

**Comment by stanislau-strelchanka** [comment]
@dcleao updated, thanks

**Comment by stanislau-strelchanka** [comment]
@dcleao, @pamval please review

**Comment by stanislau-strelchanka** [comment]
@dcleao not sure why wingman is failing. 
On my machine everything looks good.

**Pull request merged**